src/Players/AIPlayer.hs

```diff
@@ -79,10 +79,28 @@ minimize (GameTree chessBoard score children) colour = (GameTree chessBoard scor
 
 
 -- Returns the next move that maximizes the score
--- TODO: implement + test this [Yiyi]
-getMoveWithMaxScore :: GameTree -> ChessMove
-getMoveWithMaxScore gameTree = ChessMove ('z', -1) ('z', -1)
-
+-- TODO: test this [Yiyi]
+getMoveWithMaxScore :: GameTree -> ChessPieceColour -> ChessMove
```

Comment:
To test this just create a dummy GameTree with the scores already calculated and check if it returns the right move